public/presets/1/user_description.json

```diff
@@ -0,0 +1 @@
+""
```

Comment:
this file meant to be here?

Reply:
Right now, if it doesn't exist it'll error. I'll stop it from erroring and delete the file :smile: